backend/parser/build.gradle

```diff
@@ -51,6 +51,7 @@ repositories {
 
 dependencies {
     // https://mvnrepository.com/artifact/org.json/json
```

Comment:
Comment should be with its respective dependencie, removed or antlered so it fits with all dependencies